src/components/button.rs

```diff
@@ -122,6 +122,8 @@ pub struct ButtonProperties {
     pub aria_label: AttrValue,
     #[prop_or_default]
     pub aria_haspopup: AttrValue,
+    #[prop_or_default]
+    pub aria_pressed: Option<AttrValue>,
```

Comment:
this should be a `bool` type. I think the general rule is to use the type you need, with the exception of String see: [Anti-Patterns](aria-pressed)